src/Model/GroundWaterFlow/gwf3sto8.f90

```diff
@@ -312,22 +350,35 @@ subroutine sto_fc(this, kiter, hold, hnew, njasln, amat, idxglo, rhs)
       !
       ! -- set saturation used for ss
       ss_sat1 = snnew
-      ssh0 = hold(n)
-      ssh1 = DZERO
+      ssh0 = hold(n) - tp
+      ssh1 = tp
       if (this%iconf_ss /= 0) then
         if (snold < DONE) then
-          ssh0 = tp
+          ssh0 = DZERO
```

Comment:
I would change the variable to `ssb0` (see detailed explanation add at line 395)

@damianmerrick I can take care of this refactoring after the PR is merged.

Reply:
Yes, I agree. It will make more sense following your suggested changes.

@jdhughes-usgs I will leave the refactoring to you post-PR if you are happy to do that. Thanks!